pynestml/codegeneration/resources_nest/point_neuron/directives/ReadFromDictionaryToTmp.jinja2

```diff
@@ -8,7 +8,11 @@
 {%- if not variable.is_inline_expression %}
 {%- if not variable.is_state() %}
 {{declarations.print_variable_type(variable)}} tmp_{{names.name(variable)}} = {{names.getter(variable)}}();
+{%- if synapse is defined or variable in vector_symbols %}
```

Comment:
Should this condition be `variable not in vector_symbols`?

Reply:
``updateValueParam()`` does not work on vectors, hence the condition.